### ISSUE TITLE
SNAP Plot Methods Inputs

### ISSUE THREAD

**Issue by hcars** [NSSAC/CINES#219]
The UI is not very clear on one of the inputs to many of the SNAP codes that produce plots. As can be seen in the photo below, several methods require a field called _data_, but the system doesn't make it clear to the user what _data_ should be.
![plot_scc](https://user-images.githubusercontent.com/42817130/122759841-b184d080-d29a-11eb-8a13-aeb33d1991b2.jpg)

**Comment by hcars** [comment]
These all appear to have the same _data_ file input that is not explained to the user.
Name

net.science/plot_weakly_connected_component_size_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:27

Description

Plots the distribution of weakly connected component sizes

Name

net.science/plot_strongly_connected_component_size_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:26

Description

Plots the distribution of strongly connected component sizes

Name

net.science/plot_degree_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:16

Description

Plots the degree distribution

Name

net.science/plot_in_degree_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:18

Description

Plots the in degree distribution

Name

net.science/plot_out_degree_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:24

Description

Plots the out degree distribution

Name

net.science/plot_kcore_edge_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:19

Description

Plots the kcore edge distribution

Name

net.science/plot_kcore_node_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:21

Description

Plots the kcore node distribution

Name

net.science/plot_degree_ave_cluster_coeff_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:15

Description

Plots the in degree average cluster coefficient distribution

Name

net.science/plot_biconnected_component_size_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:13

Description

Plots the biconnected component size distribution

Name

net.science/plot_one_connected_component_size_distribution

Created By

dmachi

Last Updated

2021-06-15 23:20:23

Description

Plots the one connected component size distribution

**Comment by murli1201** [comment]
As these forms are handled dynamically on the UI, we show info(i) icon for the fields having 'description' property. But the 'data' does not have any description property. 

Data object has the following properties-
{match: "data*", name: "data", required: true, types: ["snap_TIntPrV"]}

**Comment by dmachi** [comment]
Yes, this will require these job definitions to get updated as well as a schema update since the input files section doesn’t currently have a “description” field.
